Notation.Plugin.AzureKeyVault/KeyVault/KeyVaultClient.cs

```diff
@@ -43,6 +45,8 @@ private record KeyVaultMetadata(string KeyVaultUrl, string Name, string Version)
         protected Lazy<SecretClient> _secretClient;
         // Error message for invalid input
         private const string INVALID_INPUT_ERROR_MSG = "Invalid input. The valid input format is '{\"contractVersion\":\"1.0\",\"keyId\":\"https://<vaultname>.vault.azure.net/<keys|certificate>/<name>/<version>\"}'";
+        // Flag for unit test
+        protected bool _testOSX;
```

Comment:
Can we remove the `_testOSX` flag? I think we can later run the unit tests using macOS host in the future PRs.

Reply:
Removed.